src/components/Images.vue

```diff
@@ -172,7 +172,11 @@ export default {
 			this.disableSwipe()
 			this.shiftX = this.shiftX + Math.round(-scrollPercX * growX)
 			this.shiftY = this.shiftY + Math.round(-scrollPercY * growY)
-			this.zoomRatio = newZoomRatio
+
+			// only change zoomRatio when multiple files are in the fileList. Disable for single shared files
+			if (this.fileList.length > 1) {
+				this.zoomRatio = newZoomRatio
+			}
```

Comment:
@max-nextcloud We can load a single file without being inline.
This is a regression

Reply:
Good point. I'll fix it.

Reply:
So i guess a good criterion will be if `el` is set.